.github/workflows/ci-actions-incremental.yml

```
@@ -115,7 +115,7 @@ jobs:
       - name: Get Date
         id: get-date
         run: |
-          echo "::set-output name=date::$(/bin/date -u "+%Y-%m")"
+          echo "date=$(/bin/date -u "+%Y-%m")" >> $GITHUB_OUTPUT
```

Are you sure about the quoting here?
```suggestion
          echo "date=$(/bin/date -u '+%Y-%m')" >> $GITHUB_OUTPUT
```
Same for all other workflow files.

---

Well, I'm not sure about the quoting, I haven't changed it :). I think it should work as bash quoting is weird.